js/src/utils/getCountriesPriceArray.js

```diff
@@ -36,25 +41,30 @@
  * //         price: 25,
  * //         currency: 'USD',
  * //     },
+ * //     {
+ * //         countries: ['BR'],
+ * //         price: 20,
+ * //         currency: 'BRL',
+ * //     },
  * ]
  * ```
  *
  * @param {Array<Object>} shippingRates Array of shipping rates in the format of `{ countryCode, rate, currency }`.
  */
 const getCountriesPriceArray = ( shippingRates ) => {
```

Comment:
💅  

The name of this file and function could be changed to groupCountriesByPriceCurrency or something similar. 
I think the current name does not describe well the function.  What do you think?

Reply:
@jorgemd24 , thanks for the fresh pair of eyes, you've got a good point! 😄 I'm thinking something like `groupShippingRatesByPriceCurrency`, will look into it. 👍

Reply:
I have renamed the function to `groupShippingRatesByPriceCurrency` in [`1c7d905` (#1190)](https://github.com/woocommerce/google-listings-and-ads/pull/1190/commits/1c7d905bdf59e20774126efec7bfa468e4357e7c).